src/main/java/org/opensearch/performanceanalyzer/writer/EventLogQueueProcessor.java

```diff
@@ -55,6 +67,9 @@
     private final long purgePeriodicityMillis;
     private final PerformanceAnalyzerController controller;
     private long lastTimeBucket;
+    private int counter;
+    private int filesCleanupPeriod;
+    private final int filesCleanupPeriodicityMillis = PluginSettings.instance().getMetricsDeletionInterval(); // defaults to 60seconds
```

Comment:
nit: Can we declare the final variables above the non-final ones?

Reply:
Done.